Client/game_sa/CEntitySA.h

```diff
@@ -236,16 +216,8 @@ class CEntitySAInterface
         using CStencilShadow_dtorByOwner = void*(__cdecl*)(CEntitySAInterface * pEntity);
         ((CStencilShadow_dtorByOwner)0x711730)(this);
     };
-
-    void DeleteRwObject()
-    {
-        using vtbl_DeleteRwObject = void(__thiscall*)(CEntitySAInterface * pEntity);
-        ((vtbl_DeleteRwObject)this->vtbl->DeleteRwObject)(this);
-    };
-
-    bool HasMatrix() const noexcept { return Placeable.matrix != nullptr; }
-
-    void RemoveMatrix() { ((void(__thiscall*)(void*))0x54F3B0)(this); }
+    
+    bool IsUsesEntityDeleteRwObject() const { return *reinterpret_cast<void**>(*reinterpret_cast<std::uint32_t const*>(this) + 0x20) == (void*)0x00534030; };
```

Comment:
What is `this + 0x20` supposed to be?
And what is the job of the `0x00534030` address?

Reply:
The class should not have this hack, removed